package.json

```diff
@@ -24,11 +24,12 @@
   "author": "Daniel Durante <me@danieldurante.com>",
   "license": "MIT",
   "dependencies": {
-    "auth-net-request": "2.2.2",
+    "auth-net-request": "SC5/authorize-net-request",
```

Comment:
yup, I believe I just updated it, but we'll still need the request dependency below